app/views/box_requests/index.html.erb

```diff
@@ -94,8 +94,7 @@
         </td>
         <td>
           <% if box %>
-            <%= box.box_items.count %> items
-            <%= box.aasm_state %>
+          <img src="https://voices-of-consent-static-images.s3.amazonaws.com/Packing+Stage.svg" title=<%= box.aasm_state %> >
```

Comment:
What do you think to use `image_tag` helper?

https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag

Reply:
That's sounds good, I will change.

Reply:
What do you think about my changes?